core/modules/taxonomy/taxonomy.module

```diff
@@ -1852,6 +1857,32 @@ function taxonomy_field_widget_form(&$form, &$form_state, $field, $instance, $la
   return $element;
 }
 
+/**
+ * Implements hook_field_widget_form_alter().
+ */
+function taxonomy_field_widget_form_alter(&$element, &$form_state, $context) {
```

Comment:
Instead of using the generic alter hook, I suggest you use `taxonomy_field_widget_options_buttons_form_alter()` and remove the check for `radios` and `checkboxes` in the if below. This hook is an implementation of `hook_field_widget_WIDGET_TYPE_form_alter()`. The widget `options_buttons` is what Backdrop uses for both radios and checkboxes.

Reply:
Additionally, doing this alter hook this way, you avoid the potential issue of messing up a custom taxonomy widget that happens to use element types `checkboxes` or `radios`. You really want to act when the site is using the widget provided by core for taxonomy selection.

Reply:
That's pretty slick, thanks. Agreed that the narrower the hook's scope is, the better.